src/pages/restore_account.vue

```
@@ -0,0 +1,268 @@
+                <div>
+                  <div class="flex justify-between items-center">
+                    <router-link
+                      to="/forgot_password"
+                      class="text-sm font-medium text-orange-500 transition-all duration-200 hover:text-orange-600 hover:underline focus:text-orange-600"
+                    >
+                      {{ t('forgot') }} {{ t('password') }} ?
+                    </router-link>
+                  </div>
+                  <FormKit
+                    id="passwordInput" type="password" :placeholder="t('password')
+                    name="password" :label="t('password')" :prefix-icon="iconPassword" :disabled="isLoading"
```

ESLint

[error] Parsing error: ',' expected.

(vue/no-parsing-error)